Core/scripts/dirac-configure.py

```diff
@@ -302,6 +311,16 @@ def forceUpdate( optionValue ):
   if newVO:
     setVO( newVO )
 
+if not extensions:
+  newExtensions = DIRAC.gConfig.getValue( cfgInstallPath( 'ExtraModules' ) )
+  if newExtensions:
+    DIRAC.gLogger.warn( "extraModules is deprecated please use extensions instead!" )
```

Comment:
Message should rather read:
ExtraModules option is deprecated, use Extensions option instead